application/src/main/java/be/jpendel/application/SignupUC.java

```diff
@@ -11,6 +11,8 @@ public SignupUC(MemberRepository memberRepository) {
         this.memberRepository = memberRepository;
     }
 
+    //TODO brrrr 'new Member' ipv een builder
+    // Volgens mijn begrip mag de applicatie laag niet een Member rechtreeks aanmaken ..waar is CreateMemberCommand ?
```

Comment:
Klopt, maar dan komt het neer op in creates:

Member newMember = memberFactory.create(createUserCommand);

en als er een actie is op member, bvb updatePassword krijgen we

Member existingMember = memberRepository.findMemberByUserName(username);
existingMember.updatePassword(command.newPassword)